src/main/java/net/mcreator/blockly/java/blocks/TextSubstring.java

```diff
@@ -50,8 +51,7 @@ else if (element.getAttribute("name").equals("text"))
 			master.processOutputBlock(to);
 			master.append("))");
 		} else {
-			master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-					"Replace text procedure block can't be empty!"));

```

Comment:
I think this was a bug from the start, maybe fix this text to have the right message in this PR too.